pyccl/tk3d.py

```diff
@@ -213,3 +216,153 @@ def get_spline_arrays(self):
             [np.exp(tk, out=tk) for tk in out]
 
         return a_arr, lk_arr1, lk_arr2, out
+
+    
+def Tk3D_SSC_Terasawa22(cosmo, deltah=0.02, 
+                     lk_arr=None, a_arr=None,
+                     extrap_order_lok=1, extrap_order_hik=1,
+                     use_log=False):
+    """ Returns a :class:`~pyccl.tk3d.Tk3D` object containing
+    the super-sample covariance trispectrum, given by the tensor
+    product of the power spectrum responses associated with the
+    two pairs of quantities being correlated. Currently this 
+    function only applicable to matter power spectrum in flat 
+    cosmology. Each response is calculated using the method 
+    developed in Terasawa et al. 2022 (arXiv:2205.10339v2) as:
+
+    .. math::
+        \\frac{\\partial P_{mm}(k)}{\\partial\\delta_L} =
+        \\left(1 + \\frac{26}{21}T_{h}(k)  -\\frac{1}{3}\\frac{d\\log P_{mm}(k)}{d\\log k}\\right)
```

Comment:
I had a quick look at Terasawa+2022 and couldn't find where the -1/3... term is coming from. If it's easy to derive or I have missed it, it'd be great to point to the Eq where it is (or that you have to use to get it) for future reference.

Reply:
The power spectrum response to the background density is consist of two components: growth response and dilation response (and change in the reference density). Terasawa+2022 focused on the growth response and the dilation term is simply the derivative of the power spectrum with respect to the wave number k. The total power spectrum response (including growth and dilation term) is found in Eq. 46 and 47 of Li, Hu, and Takada 2014a (https://arxiv.org/abs/1401.0385), for example.